provider_router.py

```
@@ -678,6 +681,24 @@ def from_env(
+        # ── Free Kimi (Moonshot) web bridge — no paid API key required ──
+        # Added near the end so it joins the chain whenever KIMI_BRIDGE_ENABLED is
+        # set. Classified free (see _FREE_CLOUD_PROVIDER_IDS) so the routing policy
+        # uses it before any paid escalation. This is what lets internal_agent /
+        # Hermes actually produce code without a paid provider.
+        try:
+            from providers.kimi_bridge import kimi_bridge_provider_config
+
+            _kimi_cfg = kimi_bridge_provider_config()
+            if _kimi_cfg is not None:
+                providers.append(_kimi_cfg)
+        except Exception as _kimi_err:  # pragma: no cover - defensive
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)